packages/components/organisms/f-cookie-banner/CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.1.3
```

Comment:
NIT – this should be a minor release rather than a patch

Reply:
Done